package.json

```diff
@@ -101,6 +101,7 @@
   "homepage": "https://github.com/uswds/uswds#readme",
   "dependencies": {
     "classlist-polyfill": "1.0.3",
+    "install": "0.13.0",
```

Comment:
What does this new dependency do? Everything else looks good.

Reply:
huh. that looks like an error